drivers/acpi/acpi.c

```diff
@@ -39,7 +39,7 @@
 
 #define AML_EXT_REGION_OP 0x80
 
-#define MAX_XSDT_HEADER_ENTRIES 20 // MUST <= the value in ABL/SBL
+#define MAX_XSDT_HEADER_ENTRIES 60 // MUST <= the value in ABL/SBL
```

Comment:
Would you mind add comment here why we need to enlarge XSDT header entries from 20 to 60?